### ISSUE TITLE
Can I use jupytext to convert Rmd to ipynb

### ISSUE THREAD

**Issue by statquant** [goerz/jupytext.vim#8]
Hello,
thanks for your plugin, I have been playing with it a bit and it is great.
I have a question regarding the usage though. I mostly code in R and find Rmd (`rmarkdown`) format superior to alternatives. However I see the advantage of being able to share `ipynb` to other people.
The question I have is then: is it possible for me to edit a Rmd file and have it "transformed" to `ipynb` in real time (basically the opposite of what I think this plugin allows to do)

EDIT:
Basically I think what I am asking is is it possible to setup a map from -> to like
{'ipynb' : 'rmarkdown', 'rmarkdown' : 'ipynb', 'md' : 'ipynb'}
Thanks and regards

**Comment by goerz** [comment]
The exact thing you're asking (have rmarkdown as the main file, and automatically have an ipynb created) is beyond the scope of the plugin. However, it is something you can do out of the box in vim:

https://stackoverflow.com/questions/4627701/how-to-automatically-execute-a-shell-command-after-saving-a-file-in-vim

You'd basically want to invoke `jupytext` whenever you save your rmarkdown file.
